diambra/arena/engine/interface.py

```diff
@@ -17,124 +17,173 @@ def __init__(self, env_address):
         self.channel = grpc.insecure_channel(env_address)
         self.stub = interface_pb2_grpc.EnvServerStub(self.channel)
 
+        # Wait for grpc server to be ready
+        print("Trying to connect to DIAMBRA Engine server ...")
+        try:
+            grpc.channel_ready_future(self.channel).result(timeout=60)
+        except grpc.FutureTimeoutError:
+            print("... failed.")
+            exceptionMessage = "DIAMBRA Arena failed to connect to the Engine Server."
+            print(exceptionMessage)
```

Comment:
No need to print all these things if we raise an exception that prints this error anyway, right?